prep-javascript-functions/main.js

```diff
@@ -0,0 +1,61 @@
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+
+var addTwoNumbersResult = addTwoNumbers(2, 5);
+console.log(addTwoNumbersResult);
+
+function convertHoursToMinutes(hours) {
+  return 4 * 60;
+}
+var convertHoursToMinutesResult = convertHoursToMinutes(4);
```

Comment:
Try multiplying `hours` times `60` so that your function works with more than just `4`.